### DISCUSSION TITLE
OSU! NEW UPDATE CAUSE crash on launch Mac Lazer

### DISCUSSION THREAD

**Discussion by NuggetNoMercy** [ppy/osu#23459] [Q&A] [answered]
I upadte OSU! Through the notification sent at the start of the old version and downloaded through Chrome, The new version will not launch and instantly crashes upon launch.
I use MacBook, and am on OSU! LAZER

Crash Explantion: The screen flashes red and closes upon launch.

I tried going through the main OSU! Download page as well and same results, I tried deleting OSU! File/Application but still the same results.

I don't know what else to do, thanks in advance.

**Comment by Rekunan** [comment]
Can you please attach logs #15188

**Comment by peppy** [comment, marked as answer]
This will likely be fixed in the next update (see https://github.com/ppy/osu-framework/pull/5780)
